create-publication-from-doi/src/main/java/no/unit/nva/doi/fetch/service/PublicationPersistenceService.java

```diff
@@ -53,7 +55,7 @@ public PublicationResponse insertPublication(Publication publication, String api
         //TODO: toResponse is exactly the same method as convertValue and will be changed upon next release
```

Comment:
I would recommend that we create a task and we put the task number in the TODO

Reply:
It is resolved now, removing comment :)